dbms/src/Flash/Coprocessor/DAGQueryBlockInterpreter.cpp

```diff
@@ -185,6 +185,16 @@ void DAGQueryBlockInterpreter::handleMockTableScan(const TiDBTableScan & table_s
         analyzer = std::make_unique<DAGExpressionAnalyzer>(std::move(names_and_types), context);
         pipeline.streams.insert(pipeline.streams.end(), mock_table_scan_streams.begin(), mock_table_scan_streams.end());
     }
+    std::vector<std::pair<UInt64, DataTypePtr>> generated_column_infos;
```

Comment:
should we do it also in `handleTableScan`?

Reply:
You mean `DAGQueryBlockInterpreter::handleTableScan`?
`DAGQueryBlockInterpreter::handleTableScan` will call DAGStorageInterpter::execute(). so no need to handle.